### PR TITLE
Update Set-CsOnlineVoiceRoute.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/9160.

### CODE REVIEW

**Comment by dariomws** [comment]
Hi @isrumnon,

Could you kindly help with a description for the BridgeSourcePhoneNumber parameter?

Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JensTrier** [comment]
@dariomws The BridgeSourcePhoneNumber parameter will be hidden, so please don't document it.

**Comment by JensTrier** [comment]
@dariomws Sorry, about the wasted update, but I was initially misinformed. The parameter should not be hidden and we should document it. Here is my input to the text:

BridgeSourcePhoneNumber is an E.164 formatted Operator Connect Conferencing phone number assigned to your Audio Conferencing Bridge. 

Using BridgeSourcePhoneNumber in an online voice route is mutually exclusive with using OnlinePstnGatewayList in the same online voice route..

When using BridgeSourcePhoneNumber in an online voice route, the OnlinePstnUsages used in the online voice route should only be used in a corresponding OnlineAudioConferencingRoutingPolicy. The same OnlinePstnUsages should not be used in online voice routes that are not using BridgeSourcePhoneNumber.

For more information about Operator Connect Conferencing, please see https://docs.microsoft.com/en-us/microsoftteams/operator-connect-conferencing-configure

**Comment by dariomws** [comment]
@JensTrier don't worry, I have added the parameter again with the description you shared.

Thanks!

**Comment by JensTrier** [comment]
@dariomws Thanks! We should also add the new parameter to New-CsOnlineVoiceRoute. Do you have time to do that or should I?

**Comment by dariomws** [comment]
@JensTrier done!

And a few additional details: -Tenant and -InMemory parameters are not available in the code anymore.

File: skype/skype-ps/skype/New-CsOnlineVoiceRoute.md [hunk 1]
```diff
@@ -65,13 +65,17 @@ This example creates a new online voice route named Route1 and populates that ro
...
 ## PARAMETERS
 
-### -Confirm
-Prompts you for confirmation before running the cmdlet.
+### -BridgeSourcePhoneNumber
+BridgeSourcePhoneNumber is an E.164 formatted Operator Connect Conferencing phone number assigned to your Audio Conferencing Bridge. Using BridgeSourcePhoneNumber in an online voice route is mutually exclusive with using OnlinePstnGatewayList in the same online voice route.
+
+When using BridgeSourcePhoneNumber in an online voice route, the OnlinePstnUsages used in the online voice route should only be used in a corresponding OnlineAudioConferencingRoutingPolicy. The same OnlinePstnUsages should not be used in online voice routes that are not using BridgeSourcePhoneNumber.
+
+For more information about Operator Connect Conferencing, please see https://docs.microsoft.com/microsoftteams/operator-connect-conferencing-configure.
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsOnlineVoiceRoute.md:73]:
```suggestion
For more information about Operator Connect Conferencing, please see [Configure Operator Connect Conferencing](https://docs.microsoft.com/microsoftteams/operator-connect-conferencing-configure).
```

File: skype/skype-ps/skype/Set-CsOnlineVoiceRoute.md [hunk 2]
```diff
@@ -61,39 +54,19 @@ PS C:\> Set-CsOnlineVoiceRoute -Identity Route1 -OnlinePstnUsages @{replace=$x}
...
+When using BridgeSourcePhoneNumber in an online voice route, the OnlinePstnUsages used in the online voice route should only be used in a corresponding OnlineAudioConferencingRoutingPolicy. The same OnlinePstnUsages should not be used in online voice routes that are not using BridgeSourcePhoneNumber.
 
-## PARAMETERS
-
-### -Confirm
-Prompts you for confirmation before running the cmdlet.
+For more information about Operator Connect Conferencing, please see https://docs.microsoft.com/microsoftteams/operator-connect-conferencing-configure.
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsOnlineVoiceRoute.md:64]:
```suggestion
For more information about Operator Connect Conferencing, please see [Configure Operator Connect Conferencing](https://docs.microsoft.com/microsoftteams/operator-connect-conferencing-configure).
```

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**